document/js-api/index.bs

```diff
@@ -334,8 +334,9 @@ A {{Module}} object represents a single WebAssembly module. Each {{Module}} obje
             1. Let |externfunc| be the [=external value=] [=external value|𝖿𝗎𝗇𝖼=] |funcaddr|.
             1. [=Append=] |externfunc| to |imports|.
         1. If |externtype| is of the form [=𝗀𝗅𝗈𝖻𝖺𝗅=] |globaltype|,
-            1. If |globaltype| [=Type=](|v|) is [=Number=],
+            1. If [=Type=](|v|) is [=Number=],
                 1. If |globaltype| is [=𝗂𝟨𝟦=], throw a {{LinkError}} exception.
+                1. If |globaltype|.<em>[=global type|mut=]</em> is [=var=], throw a {{LinkError}} exception.
```

Comment:
A globaltype isn't a record, so this doesn't quite work. Related, the previous line is already off because it equates a globaltype with a value type. Instead, you have to use pattern matching like "Let "mut t" be |globaltype|".

Reply:
Instead of inspecting mutability and throwing manually, it might be more modular to just go on creating the global instance, but with a globaltype that is always "const t". Then the core semantics will take care of the check.

Reply:
> A globaltype isn't a record, so this doesn't quite work.

Yes, I noticed this too, but I wanted to be consistent with what was already there. I guess we should fix the other upstream, see https://github.com/WebAssembly/spec/issues/774.